src/containers/permissionssummary/PermissionsSummaryEpic.js

```diff
@@ -87,6 +166,60 @@ function createAclsObservables(entitySetId, properties) {
 
 
 /* EPICS */
+function getOrgsMembersEpic(action$, store) {
+  return action$
+    .ofType(orgsActionTypes.FETCH_ORGS_SUCCESS)
+    .map((action) => {
+      const orgs = store.getState().getIn(['organizations', 'organizations']);
+      const orgIds = action.orgIds || orgs.toArray().map(entry => entry[0]);
```

Comment:
Instead of `orgs.toArray().map(entry => entry[0])`, you can do `orgs.keySeq().toJS()`.